contracts/hardhat.config.js

```diff
@@ -179,6 +187,13 @@ module.exports = {
       live: true,
       saveDeployments: true,
     },
+    sonic: {
+      url: sonicProviderUrl,
+      accounts: [
```

Comment:
needs to have `live` and `saveDeployments` params set to true

Reply:
good catch thanks